incuna_auth/middleware/permission_feincms.py

```diff
@@ -53,7 +53,10 @@ def _get_resource_access_state(self, request):
 
         # Chase inherited values up the tree of inheritance.
         INHERIT = AccessState.STATE_INHERIT
-        while feincms_page.access_state == INHERIT and hasattr(feincms_page, 'parent'):
+        while (
+            feincms_page.access_state == INHERIT and
+            getattr(feincms_page, 'parent', None)
```

Comment:
Other than testing?  I don't know.  I'm not sure I want to find out in practice :P  I'll have a look.